docs/README.md

```diff
@@ -39,16 +39,15 @@ systems, but all the principles apply equally to such scenarios.
 The image below shows the modules structure of a single SIP adapter.
 To provide a level of flexibility, SIP splits the problem into two, giving the possibility to develop and/or deploy connectors
 for individual systems separately. The integration
-logic is divided into three modules: **System A - Connector**, **System B - Connector** and **Domain**. The fourth module is the **Application**
-module. Its purpose is to converge the three integration modules into one deployable and executable application.
-The blue arrows represent dependencies of the application module.
+logic is divided into three packages: **System A - Connector**, **System B - Connector** and **Common - Domain**.
 
 ![Image of SIP Adapter](./img/SIP_readme_adapter_detail.svg?raw=true "SIP Adapter")
 
 **Domain**
 
-The **Domain** module takes a central position in the image, because it provides the common data model for both systems.
-Also, it separates the system connectors from each other to allow loose coupling between them. It should not contain any
+The **Common** Package takes a central position in the image, 
+because it provides the common data model inside domain for both systems and common util functionalities.
+Also, it separates the system connectors from each other to allow loose coupling between them. Domain package should not contain any
```

Comment:
'Also, it separates the system connectors from each other to allow loose coupling between them.' - not true anymore, it does not separate anything and does not give loose coupling.

The remaining text is true, but I'm afraid it becomes hard to understand in current format. The sub title is Common, and we start talking about common package than all of a sudden the story switches to domain package.